### ISSUE TITLE
[Feature request] Actual values showed for all graphs

### ISSUE THREAD

**Issue by AseKarlsson** [kalkih/mini-graph-card#588]
Hi,
I am am quite new to both HA and Github, so maybe I post this in the total wrong forum.
I love the mini-graph-card but have an improvement request.
This is the view now with only one of the values of the graph showing:
![image](https://user-images.githubusercontent.com/80900477/114213726-a68cf600-9963-11eb-8c4a-986457d0a43d.png)
Can something like this be done (Photoshopped):
![image](https://user-images.githubusercontent.com/80900477/114214392-8dd11000-9964-11eb-856b-fb7d4cbcfcee.png)
Or like this then even more than 2 graphs would be possible:
![image](https://user-images.githubusercontent.com/80900477/114214880-30898e80-9965-11eb-9264-80e4bc9f45ed.png)

Thanks,
Ase

**Comment by kalkih** [comment]
Hello,
This can already be achieved by utilizing the `show_state` [entity option](https://github.com/kalkih/mini-graph-card#entities-object). First entity in the config will always show by default.

```yaml
...card config
entities:
  - entity: sensor.entity_1
  - entity: sensor.entity_2
    show_state: true
```

**Comment by AseKarlsson** [comment]
Great, Thanks!

**Comment by AseKarlsson** [comment]
Hi,
Now I added a 3rd sensor to the a graph and the result is like the picture below.
Tried to read the readme and tested a lot of different setting but could not figure out if this is doable.

1. Can I have the same font size on all 3 values?
2. Can I move the 3rd value (31.7) to the center so it would be easier to understand what value is which as the description below?
3. Can the temperature values have the same color as the graph to help?

Thanks,
Ase

![Heating](https://user-images.githubusercontent.com/80900477/127190988-f7141c07-2e04-4bc2-90f4-856991dbb1e6.jpg)
